claim/gql_mutations.py

```diff
@@ -258,6 +262,23 @@ def create_file(date, claim_id, document):
     return file_path
 
 
+def handle_claimdoc_np_attachment(data):
+    url_template = 'https://claimdoc.hib.gov.np/upload_documents?'
+    token = os.environ.get("CLAIMDOC_TOKEN", default='testToken')
+    hf_id = None
+    if "claim_id" in data:
+        hf_id = Claim.objects.get(validity_to__isnull=True, id=data["claim_id"]).health_facility.id
+    else:
+        claim = Claim.objects.get(validity_to__isnull=True, uuid=data["claim_uuid"])
+        data["claim_id"] = claim.id
+        hf_id = claim.health_facility.id
+    url_params = {'claim_code': data["claim_id"], 'token': token, 'hf_id': hf_id}
+    return requests.get(url_template, url_params).url
+
+
+attachment_strategies = {"claimdoc": handle_claimdoc_np_attachment}
+
+
```

Comment:
What would be the purpose of such change?

Reply:
Signal enable loose coupling between module, e.i. the claim module does not need to be aware at initialization of the handlers; handlers can be denied in a "customization" module easing the system updates (no forks)

Reply:
At least rename the function handle_claimdoc_np_attachment to handle_claimdoc_attachment
and put the URL in the config

Reply:
I rewrote this completely - now you have to create a new file in attachment_strategies directory with function called "handler" inside. Thanks to this there should be almost no conflicts in the code itself - additional file is needed, so git can handle it easily. 

What is still an unresolved problem (that I suggest we handle later) is changing attachment_strategies dictionary, since we have to extend it manually, but apart from this, no changes to gql_mutations.py are needed.